### PR TITLE
adding required claims to error message

### PR DESCRIPTION
**Pull request by jennyf19**
@henrik-me asked that we include the missing claims in the log message

### CODE REVIEW

File: msal/src/Microsoft.Identity.Client/TokenCache.cs [hunk 1]
```diff
@@ -276,7 +276,7 @@ private bool ContainsClaimsRequiredToStoreInCache(IdToken idToken, RequestContex
                 }
                 else
                 {
-                    msg = "Skipping storing tokens in the cache - IdToken does not contain required claims";
+                    msg = "Skipping storing tokens in the cache - IdToken does not contain required claims of TenantId or PreferredUsername.";
```
Comment (henrik-me) [inline msal/src/Microsoft.Identity.Client/TokenCache.cs]: >of TenantId or PreferredUsername [](start = 107, length = 32)

we should use the actual names used in the claims. ideally we have const somewhere with these in that we can use. (the const is where the cache is read/persisted)

Comment (jennyf19) [reply msal/src/Microsoft.Identity.Client/TokenCache.cs]: done. thx. ⛄️ #!

**Review by jmprieur — APPROVED** [review]
LGTM

**Comment by henrik-me** [comment]
We should also add a link to the wiki page talking about work arounds and issues.

looks great

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**